target/linux/ath79/generic/base-files/etc/board.d/01_leds

```diff
@@ -184,6 +184,11 @@ comfast,cf-e314n-v2)
 	ucidef_set_led_rssi "rssimediumhigh" "RSSIMEDIUMHIGH" "green:rssimediumhigh" "wlan0" "51" "100"
 	ucidef_set_led_rssi "rssihigh" "RSSIHIGH" "green:rssihigh" "wlan0" "76" "100"
 	;;
+comfast,cf-e355ac-v2)
+	ucidef_set_led_netdev "lan" "LAN" "green:lan" "eth0"
+	ucidef_set_led_netdev "wlan2g" "WLAN2G" "blue:wlan2g" "phy1tpt"
+	ucidef_set_led_netdev "wlan5g" "WLAN5G" "red:wlan5g" "phy1tpt"
```

Comment:
These can be dropped since the same binding are already defined in the dts.

Reply:
Removed